file-formats/dobj/type/zif_aff_dobj_v1.intf.abap

```diff
@@ -0,0 +1,61 @@
+INTERFACE zif_aff_dobj_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Structure Definition</p>
+    "! Structure Definition
+    BEGIN OF ty_structure_definition,
+      "! <p class="shorttext">Structure Definition</p>
+      "! Parent Table
+      "! $required
+      record_no       TYPE n LENGTH 4,
+      "! <p class="shorttext">Parent Table</p>
+      "! Parent Table
+      "! $required
+      parent_table    TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Dependent table</p>
+      "! Dependent table
+      "! $required
+      dependent_table TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_structure_definition,
+    "! <p class="shorttext">Structure Definitions</p>
+    "! Structure Definitions
+    ty_structure_definitions TYPE SORTED TABLE OF  ty_structure_definition WITH UNIQUE KEY record_no.
+
+
+  TYPES:
+    "! <p class="shorttext">Data destruction object</p>
+    "! $required
+    BEGIN OF ty_dobj_details,
+      "! <p class="shorttext">Destruction Program</p>
+      "! Destruction Program
+      "! $required
+      destr_program        TYPE zif_aff_types_v1=>ty_description_60,
+      "! <p class="shorttext">Application Component</p>
+      "! Application Component
+      applic_component     TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Structure Definition</p>
+      "! Structure Definition
+      structure_definition TYPE ty_structure_definitions,
+    END OF ty_dobj_details.
+  TYPES:
+    "! <p class="shorttext">Data Destruction Object</p>
+    "! ILM destruction object (DOBJ) v1
+    BEGIN OF ty_main,
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE zif_aff_types_v1=>ty_header_60_cloud,
+      "! <p class="shorttext">Description</p>
+      "! Description
+      "! $required
+      description    TYPE zif_aff_types_v1=>ty_description_60,
```

Comment:
The `header` has already defined a field `description`. Do you need two descriptions?

Reply:
Does the description in the header correspond to the description of the data destruction object? The data destruction object is user defined name.

Reply:
> Does the description in the header correspond to the description of the data destruction object?
Yes, this is the idea.

> The data destruction object is user defined name.
I guess I don't get your point here. How is the name of the destruction object related to the field `description`?